projects/ngrx.io/content/examples/store/src/app/app.module.ts

```diff
@@ -6,13 +6,17 @@ import { AppComponent } from './app.component';
 
 // #docregion imports
 import { StoreModule } from '@ngrx/store';
-import { counterReducer } from './counter.reducer';
+import { counterFeature } from './counter.reducer';
 // #enddocregion imports
 import { MyCounterComponent } from './my-counter/my-counter.component';
 
 @NgModule({
   declarations: [AppComponent, MyCounterComponent],
-  imports: [BrowserModule, StoreModule.forRoot({ count: counterReducer })],
+  imports: [
+    BrowserModule,
+    StoreModule.forRoot({}),
+    StoreModule.forFeature(counterFeature),
```

Comment:
Does it make sense to update `StoreModule.forRoot` to take a `Feature` as an argument?